src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -84,8 +98,10 @@ function EpidemicModelProvider({ children }: EpidemicModelProviderProps) {
     stateCode: "US Total",
     countyName: "Total",
     countyLevelDataLoading: true,
-    rateOfSpreadFactor: 3.7,
+    rateOfSpreadFactor: "high",
     usePopulationSubsets: false,
+    facilityOccupancyPct: 1,
```

Comment:
I haven't checked the calculation logic using these downstream yet, but at first glance it seems wrong for these two facility housing type percentages to sum to greater than 1. Is that an issue?

Reply:
Ah sorry I thought these were two dueling types of housing, but I see that they are orthogonal. One is overall occupancy, the other is the split between housing types. Don't mind me, but leaving this comment for any other reviewers who may have made the same initial mistake.